test/e2e/authorization.go

```diff
@@ -23,19 +23,53 @@ var (
 		{"CollaboratorPowerUser"},
 		{"Administrator"},
 	}
+
+	// "secrets" are not included as they have their own set of test cases.
+	namespacedResources = []string{
+		"pods",
+		"apps/deployments",
+		"apps/statefulsets",
+		"apps/deployments/scale",
+		"apps/statefulsets/scale",
+		"services",
+		"persistentvolumes",
```

Comment:
This is a global resource IIRC

Reply:
Hmm, good point. I'll remove it from the list.

Reply:
I moved `persistentvolumes` to `globalresources` list in the next commit.

Reply:
The tests have started failing now. Do we actually not allow write access to `persistentvolumes` to collaborators?

Reply:
We do allow write access here: https://github.com/zalando-incubator/kubernetes-on-aws/blob/0b3171b3f858587dc09eb37e33e451bcb69ef68a/cluster/manifests/roles/poweruser-role.yaml#L27-L45

Reply:
The way the collaborator role is defined is that it's assumed you also have the PowerUser/engineer role for all the poweruser permissions it's not a superset of poweruser by itself.

We only give the collaborator role these specific permissions: https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/cluster/manifests/roles/collaborator-roles.yaml

Reply:
ah, that makes sense. I'll remove the `persistentvolumes` from the global resources then.